internal/cli/atlas/backup/compliancepolicy/copyprotection.go

```diff
@@ -42,7 +42,9 @@ const (
 	active  = "ACTIVE"
 )
 
-var copyProtectionTemplate = `Copy protection has been set to: {{.CopyProtectionEnabled}}`
+var copyProtectionTemplate = `
```

Comment:
any reason for the extra new line at the start?

Reply:
removed